src/core/logical_layers/rfc.md

```diff
@@ -0,0 +1,73 @@
+# Layer types
+
+Before answering the question what are logical layers, we should first consider what kind of layers we have in the context of a map.
+
+## Map Style Layers
+
+The [Style Specification](https://maplibre.org/maplibre-gl-js-docs/style-spec/) defines a map layer as a way of representing data.
+Such a layer is part of the map style and can be of type "fill", "line", "symbol", "circle" and so on.
+In the context of our application, we refer to such layers as map style layers, or just a style layer
+
+## Sources of layers
+
+If we divide layers by data delivery format, we can identify "vector", "raster" and "geojson" layers.
+However, this is a simplification - despite the fact that it directly affects the way the layer style is described, this division still refers to _layer sources_.
+At the same time, the vector data source type also has layers, which we call source-layer
+
+## Logical layers
+
+And, at the top of the list are the logical layers. A logical layer is a concatenation of several data and style layers linked together by a common theme.
+
+This is how these layers look like a tree:
```

Comment:
```suggestion
Here's a tree structure example of a logical layer
```

Reply:
How about "This is how these layers nested" ?

Reply:
Yes!